tests/transforms/test_hamiltonian_expand.py

```diff
@@ -404,6 +404,18 @@ def test_constant_offset_no_grouping(self):
         processed_res = fn(dummy_res)
         assert qml.math.allclose(processed_res, 10.0)
 
+    def test_only_constant_offset(self):
+        """Tests that hamiltonian_expand can handle a single Identity observable"""
+
+        H = qml.Hamiltonian([1.5, 2.5], [qml.I(), qml.I()])
+
+        @functools.partial(qml.transforms.hamiltonian_expand, group=False)
```

Comment:
What would be required to allow `group=True` in that case?

Reply:
For `hamiltonian_expand`, if `group=True`, it obtains the grouping information from `LinearCombination.compute_grouping()`, which actually does not work when the hamiltonian **only** consists of `Identity` with no wires. I could open an issue for this edge case bug but I'm not sure if it's worth trying to fix it right now.

Reply:
Ah okay when its really just the case where it is _only_ Identities than it seems indeed like an edge case. I thought it didnt work whenever there is more than one identity in there 👍